VERSIONING.md

```diff
@@ -57,16 +69,16 @@ The Serverless Framework supports the major cloud providers Node.js runtime vers
 
 1. Is it okay to mark a feature as deprecated in version 1.4.0 and then remove it in 1.8.0
 
-No, since this is a breaking change it should trigger a major version bump to 2.0.0
+Yes, a breaking change to individual features may happen with a MINOR release
 
-2. Can we change everything in a major version bump?
+1. Can we change everything in a major version bump?
 
-Yes, this is the purpose of major version bumps. Ideally every breaking change has a clear and well documented migration path. In best case the features were already introduced earlier and upgrading is not a dealbreaker.
+Yes, a new major will most likely announce a completely new (or significantly changed) framework architecture
 
-3. Can we do a major version bump without a breaking change?
+1. Can we do a major version bump without a breaking change?
 
-No, as we strictly follow Semantic Versioning. The suggested strategy is to add features with minor releases and only do major version bumps when we take out deperecated features. Sometimes this is not possible, but as suggested above then a well documented migration path should come with the release.
+No, it won't happen
 
-4. Why is CLI output a breaking change?
+1. Why is CLI output a breaking change?
 
 Right now we don't provide an option to output a well formated datastructure for a CLI command. Once we have such an option the default CLI output will not be part of the breaking changes list, but rather the datastructure. Also to note here if we add to that datastructure it will not be a breaking change. If we remove or change something from that datastructure it is a breaking change.
```

Comment:
After reading this again I think it doesn't really make sense.

[Above](https://github.com/serverless/serverless/pull/6195/files#diff-baa362176b63e6912fc6721235a68502L45) we state that changing CLI `stdout` is not a breaking change. However here we say that it would be a breaking change until we introduce a formatted datastructure.

Isn't this a contradiction? Or am I missing something? 🤔

Reply:
This was left intact, but I understand it as:

- Why is change of CLI output a breaking change?
- It's because we do not provide a programmatically consumable alternative. Once we will provide it, we will categorize changes to human readable CLI output as non-breaking

Still I see it as controversial, as anyway I think we improve wording and fix typos of CLI output now, and do not announce such changes as breaking. So technically we do not perfectly comply to this

Reply:
Yes, true. I just stumbled upon this while reading it. It's very vague, up for interpretation and controversial.

Let's keep it for now (as nobody complained so far) and update it later on.